src/COMPONENTS/PRODUCTCARD/Productcard.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable */
+import { Link } from "react-router-dom";
+
+const Productcard= (props) => {
+  const { id, title, price, image} = props;
+  return (
+    <Link to={`/product/${id}`}>
+      <div className="card" id={id}>
+        <div className="card-container">
+          <img className="card-img" src={image} width="200px" height="200px" />
+          <div className="card-details">
+            <h1 className="card-title">{title}</h1>
+            <p className="card-price">$ {price}</p>
+          </div>
+          
+       </div>
+      </div>
+    </Link>
+  );
```

Comment:
For the home page design:
- [ ] Please make the width of the left product card equal to the width of the right product card.
- [ ] Kindly change the color of the text below the product image to make it appear properly and try to stick as much as possible to the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/189378450-26439979-5fd8-4232-be10-fab1e2745cf5.png)  | ![image](https://user-images.githubusercontent.com/68796232/189378434-da267aa2-25f8-42fd-8bc5-fde0959adcf9.png)  |

Reply:
thank you ,change made